config/samples/airlock_v1alpha1_mongodbcluster.yaml

```diff
@@ -35,6 +35,9 @@ spec:
   # Optional. Append this prefix to all default/generated usernames for this cluster. Will be ignored if "username" is already set on the access request.
   userNamePrefix: test-use1-
 
+  # Optional. If this is set, along with useAtlasApi, all the kubernetes nodes on the cluster will be added to the Atlas firewall, using the rke.cattle.io/external-ip annotation.
+  allowOnAtlasFirewall: true
+
```

Comment:
I know right now using rancher.  But thinking might be useful to go ahead and use a property here with the node public ip style?  So that we are already specifying rancher annotation so later on when we add another maybe it can even become default with out disrupting us